README.md

```diff
@@ -198,10 +198,16 @@ If the `source` block has the optional key `gchat_webhook`, then it will also se
 
 - `state`: The state to set. One of `error`, `failure`, `pending`, `success`, `abort`. Subject to the mapping explained in section [Effects](#effects).
 
-## Optional params
+## Optional params for GitHub commit status
```

Comment:
I think you should also update the note here: https://github.com/Pix4D/cogito#note

> `The put step requires one and only one ["put inputs"](https://concourse-ci.org/put-step.html#put-step-inputs) to be specified, otherwise it will error out. For example:`

Reply:
Also, this makes me wonder:
Should the input be added to required for put step (even now, before your PR)
With current `cogito` if one does not set `inputs: [the-repo]` the resource will fail and this should be added under required for put step. I now that this is not a params, but it should not be under note. 

Also, as I wrote above, note is not up to date with this PR anymore.

Reply:
You are right, I will update the README.

> With current cogito if one does not set inputs: [the-repo] the resource will fail and this should be added under required for put step. I now that this is not a params, but it should not be under note.

I see what you mean. Somehow Concourse put.inputs are like global variables: you don't know if and where they will be used in the job :-/ I will see how to clarify.

Reply:
Updated.